src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -63,9 +63,275 @@ private MobileAdsDataProvider() {
                 "\"s2\":\"article\"",
                 "\"dmn\":\"wikiacom\"",
                 "\"hostpre\":\"",
+                "\"skin\":\"mercury\"",
                 "\"wpage\":\"synthetictests/dfpparams\"",
                 "\"ref\":\"direct\"",
                 "\"lang\":\"en\"",
+                "\"skin\":\"mercury\"",
+                "\"esrb\":\"teen\""
+
+            ),
+            Arrays.asList(
+                "\"pos\":\"MOBILE_TOP_LEADERBOARD\"",
+                "\"src\":\"mobile\""
+            )
+        }
+    };
+  }
+
+  @DataProvider
+  public static Object[][] dfpParams() {
+    return new Object[][]{
+        {
+            "yugioh",
+            "Dark_Magician",
+            "wka.ent/_yugioh//article",
+            "MOBILE_TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"ent\"",
+                "\"s0v\":\"comics\"",
+                "\"s0c\":[\"gaming\",\"anime\",\"videogames\"]",
+                "\"s1\":\"_yugioh\"",
+                "\"s2\":\"article\"",
+                "\"dmn\":\"wikiacom\"",
+                "\"hostpre\":",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"hostpre\":" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)